packages/fast-components-react-base/src/select/select.tsx

```diff
@@ -166,11 +168,10 @@ class Select extends Foundation<SelectHandledProps, SelectUnhandledProps, Select
                 onClick={this.handleClick}
                 onFocus={this.handleFocus}
                 tabIndex={-1}
-                role="listbox"
+                role="combobox"
```

Comment:
What is the justification for making this a `combobox`? https://www.w3.org/TR/wai-aria-1.1/#combobox establishes a combobox as a textbox with autosuggest functionality - isn't select an implementation of the native `select` element?

Reply:
Native selects are announce as a combobox.  There are some definite inconsistencies between native implementations and aria spec.

Reply:
It appears that ARIA is moving their implementation guidance towards the native select: http://w3c.github.io/aria/#combobox, so I'm supportive of this change.

Reply:
Can we audit this control to ensure we're meeting the latest standards linked to above? If we're going to change the role here it's the best time to make sure the control accurately reflects the above draft.

Reply:
I've messed with it some and I have not been able to come up with a better behavior than what we have.  Biggest issue I see now is that narrator calls it a "ComboEdit" instead of a "ComboBox".  My recommendation is that we go with this and create a task to revisit when aria 1.2 is implemented and we can tweak over to something closer to this (I don't think we'll be too far off) - https://github.com/w3c/aria/wiki/Resolving-ARIA-1.1-Combobox-Issues

Reply:
Sounds good - thanks for looking deeper into this. Can you link to the issue when you get it written up? Thx!

Reply:
Here you go - https://github.com/microsoft/fast-dna/issues/2454

Reply:
**Current behavior**
-	Narrator+UIA: “Zoom, 100%, 8 of 17, selected”
-	NVDA+IA2: “Zoom  list  collapsed  100%, 100%  8 of 17”

I think this is a solution we should seriously: [https://www.w3.org/TR/wai-aria-practices-1.1/examples/listbox/listbox-collapsible.html
](https://www.w3.org/TR/wai-aria-practices-1.1/examples/listbox/listbox-collapsible.html)

_“The following example implementation of the design pattern for listbox demonstrates a collapsible single-select listbox widget that is functionally similar to an HTML select input with the attribute size="1"”_ 

**Proposed current behavior**
-	Narrator+UIA: “Choose an element: Neptunium, menu, has popup”
-	NVDA+IA2: “Choose an element: Neptunium, menu button  submenu”